### PR TITLE
Feature/payment

### PR DESCRIPTION
**Pull request by HASH10DEV**
please verify

### CODE REVIEW

File: deployments/payment/helm/.helmignore [hunk 1]
```diff
@@ -0,0 +1,23 @@
+# Patterns to ignore when building packages.
```
Comment (kishalayb18) [inline deployments/payment/helm/.helmignore]: remove this file

File: deployments/payment/helm/templates/_helpers.tpl [hunk 2]
```diff
@@ -0,0 +1,78 @@
+{{/*
+Resources of the Container
+*/}}
+{{- define "container.resources" -}}
+{{- if .Values.containers.resources -}}
+resources:
+  limits:
+    cpu: {{ (.Values.containers.resources.limits).cpu | default "500m" }}
```
Comment (kishalayb18) [inline deployments/payment/helm/templates/_helpers.tpl:8]: compare the CPU memory with the main repository

File: deployments/payment/helm/templates/deployment.yaml [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+      {{if .Values.psp.enabled }}
+      serviceAccountName: {{ .Values.psp.serviceAccountName }}
+      {{end}}
+      containers:
+      - name: {{ .Values.appName }}
+        image: {{ .Values.containers.image.repo }}:{{ .Values.containers.image.tag }}
+        imagePullPolicy: {{ .Values.containers.image.pullPolicy }}
+        # agent networking access
+        env:
+          - name: INSTANA_AGENT_HOST
```
Comment (kishalayb18) [inline deployments/payment/helm/templates/deployment.yaml:26]: compare the env with the main repository

File: deployments/payment/helm/templates/tests/test-connection.yaml [hunk 4]
```diff
@@ -0,0 +1,15 @@
+apiVersion: v1
```
Comment (kishalayb18) [inline deployments/payment/helm/templates/tests/test-connection.yaml]: why is this file required

File: deployments/payment/helm/values.yaml [hunk 5]
```diff
@@ -0,0 +1,27 @@
...
+      memory: 100Mi
+    requests:
+      cpu: 100m
+      memory: 50Mi
+
+service:
+  type: NodePort
+  ports:
+    name: http
+    port: 8080
```
Comment (kishalayb18) [inline deployments/payment/helm/values.yaml:26]: compare the port numbers

**Review by kishalayb18 — CHANGES_REQUESTED** [review]
please review the suggested changes

File: deployments/payment/helm/.helmignore (hunk 1 above)
Comment (HASH10DEV) [reply deployments/payment/helm/.helmignore]: resolved

**Comment by kishalayb18** [comment]
restructure the helm folder

**Review by kishalayb18 — APPROVED** [review]
LGTM

**Pull request merged**